### ISSUE TITLE
Upgrade from 1.7.6.3 to 1.7.6.5 failed Following PrestaShop upgrade documentation

### ISSUE THREAD

**Issue by fmipsylone** [PrestaShop/PrestaShop#18961]
#### Describe the bug

Everything is in the title.

```
PHP Fatal error:  Class 'Datas' not found in /var/www/install/init.php on line 73
PHP Stack trace:
PHP   1. {main}() /var/www/install/upgrade/upgrade.php:0
PHP   2. require_once() /var/www/install/upgrade/upgrade.php:52
```

#### Steps to Reproduce

Steps to reproduce the behavior following documentation https://devdocs.prestashop.com/1.7/basics/keeping-up-to-date/upgrade/:

1. `wget -O prestashop-upgrade.zip https://www.prestashop.com/download/old/prestashop_1.7.6.5.zip`
2. `mkdir upgrade1.7.6.5`
3. `cd upgrade1.7.6.5`
4. `unzip ../prestashop-upgrade.zip && unzip prestashop.zip`
5. List of PrestaShop files seems OK
6. `cp -R /var/www/ /var/www_1.7.6.3` (to create a copy of the previous website)
7. Switch to PrestaShop to maintenance mode from BO
8. `cp -R upgrade1.7.6.5/* /var/www/`
9. `php /var/www/install/upgrade/upgrade.php`
9. `PHP Fatal error:  Class 'Datas' not found in /var/www/install/init.php on line 73`

Following this errors I checked my website:
My website BO seems to work correctly, but the front don't and all the products do not appears on BO > Catalog > Products... Whereas they are still well present in database...

Rollback to my previous installation folder is OK. The Front is working and Products appears correctly on the BO...

#### Additional information

* PrestaShop version: 1.7.6.3
* PHP version: 5.6

**Comment by sarahdib** [comment]
Hello,

This issue is a duplicate of #18792 

Thanks to follow the first issue, I will close this one.

**Comment by sarahdib** [comment]
Duplicate of #18792

**Comment by fmipsylone** [comment]
Hello @sarahdib 
I searched before publishing my issue. I did not find one concerning this one. And I do not find the related issue in #18792 from my duplicated. To be able to communicate, Can you tell me which one it is ?
Thank you

**Comment by bm-services** [comment]
Hello,
We reproduce this bug.
I aggree with you, I think this bug are not related with #18792 but with #19557.
It's an important bug because we can't upgrade PrestaShop versions < 1.7.6.5 to versions 1.7.6.5 and superior...

**Comment by matks** [comment]
> Hello,
> We reproduce this bug.
> I aggree with you, I think this bug are not related with #18792 but with #19557.
> It's an important bug because we can't upgrade PrestaShop versions < 1.7.6.5 to versions 1.7.6.5 and superior...

If I am not wrong, the code is fixed in 1.7.6.6 by this PR https://github.com/PrestaShop/PrestaShop/pull/19010.

We will unfortunately not be able to fix versions 1.7.6.0 to 1.7.6.5 because they are already released and it would be very bad to modify a release already public.

**Comment by bm-services** [comment]
Ok ! I had not found PR #19010.

Thanks for information !
